TypeScript build fails because dompurify is imported but not installed.

## User Request

**Type:** bug
**Target:** Console Application
**Submitted by:** @rishi-jat
**Console Request ID:** 52dd7631-3da7-4f12-87aa-494268556cfc

## Description

**What happened**
`npm run build` fails with TS2307: Cannot find module 'dompurify'.

**What I expected**
All imported packages should be installed and build should succeed.

**Steps to reproduce**
1. cd web
2. npm run build
3. Observe error

**Proof**
```
src/components/cards/ClusterLocations.tsx(8,23): error TS2307: Cannot find module 'dompurify' or its corresponding type declarations.
```

**Impact**
Build fails. Cannot deploy production code.

**Root cause (code)**
- File: web/src/components/cards/ClusterLocations.tsx
- Line 8: `import DOMPurify from 'dompurify'`
- Package not in package.json dependencies

---
*This issue was automatically created from the KubeStellar Console.*

--- a/pkg/agent/provider_antigravity.go
+++ b/pkg/agent/provider_antigravity.go
@@ -80,8 +80,26 @@ func (a *AntigravityProvider) Refresh() {
 // Handshake verifies that the Antigravity CLI is installed and can respond.
 // It returns a structured result with prerequisites and actionable errors.
 func (a *AntigravityProvider) Handshake(ctx context.Context) *HandshakeResult {
-	// Re-detect CLI in case the user installed it after the server started.
-	a.detectCLI()
+	// If a cliPath is already configured, validate it exists on disk.
+	// Otherwise, re-detect CLI in case the user installed it after the server started.
+	if a.cliPath != "" {
+		if _, err := os.Stat(a.cliPath); err != nil {
+			return &HandshakeResult{
+				Ready:   false,
+				State:   "failed",
+				Message: fmt.Sprintf("Configured CLI path does not exist: %s", a.cliPath),
+				CliPath: a.cliPath,
+				Prerequisites: []string{
+					fmt.Sprintf("Verify the CLI exists at %s", a.cliPath),
+					"Install the Antigravity CLI (npm install -g @anthropic-ai/antigravity or download from the project page)",
+					"Ensure the 'antigravity' binary is in your PATH",
+					"Restart kc-agent after installing",
+				},
+			}
+		}
+	} else {
+		a.detectCLI()
+	}
 
 	if a.cliPath == "" {
 		return &HandshakeResult{

--- a/pkg/api/handlers/user.go
+++ b/pkg/api/handlers/user.go
@@ -44,7 +44,7 @@ func (h *UserHandler) UpdateCurrentUser(c *fiber.Ctx) error {
 	// Only allow updating certain fields
 	var updates struct {
 		Email   string `json:"email"`
-		SlackID string `json:"slack_id"`
+		SlackID string `json:"slackId"`
 	}
 	if err := c.BodyParser(&updates); err != nil {
 		return fiber.NewError(fiber.StatusBadRequest, "Invalid request body")

--- a/pkg/k8s/workload_scaling_test.go
+++ b/pkg/k8s/workload_scaling_test.go
@@ -67,12 +67,26 @@ func TestScaleWorkload(t *testing.T) {
 }
 
 func TestDeleteWorkload(t *testing.T) {
+	deployObj := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "apps/v1",
+			"kind":       "Deployment",
+			"metadata": map[string]interface{}{
+				"name":      "dep1",
+				"namespace": "default",
+			},
+			"spec": map[string]interface{}{
+				"replicas": int64(1),
+			},
+		},
+	}
+
 	scheme := runtime.NewScheme()
 	gvrMap := map[schema.GroupVersionResource]string{
 		{Group: "apps", Version: "v1", Resource: "deployments"}: "DeploymentList",
 	}
 
-	fakeDyn := fake.NewSimpleDynamicClientWithCustomListKinds(scheme, gvrMap)
+	fakeDyn := fake.NewSimpleDynamicClientWithCustomListKinds(scheme, gvrMap, deployObj)
 
 	m, _ := NewMultiClusterClient("")
 	m.dynamicClients["c1"] = fakeDyn

--- a/web/public/mockServiceWorker.js
+++ b/web/public/mockServiceWorker.js
@@ -1,4 +1,4 @@
- 
+/* eslint-disable */
 /* tslint:disable */
 
 /**

